### DISCUSSION TITLE
Archive page not working as intended.

### DISCUSSION THREAD

**Discussion by MassGaming25** [xenocrat/chyrp-lite#193] [Q&A] [answered]
Hey there, I recently started using chyrp-lite and most of it works out of the box. The only problem i encountered is that i get an error when trying to visit the archive page on my website. Could any of you help me?

The exact error message on the website:
![image](https://github.com/xenocrat/chyrp-lite/assets/102473883/b00254fa-475e-4e6c-8af4-1ae48079462c)

**Comment by xenocrat** [comment, marked as answer]
Hello there,

This is caused because your PHP installation does not have the [multi-byte string extension](https://www.php.net/manual/en/book.mbstring.php) installed. However Chyrp Lite should be checking for this before attempting to use it. Can you enable DEBUG mode by following [these instructions](https://github.com/xenocrat/chyrp-lite/wiki/Debug-and-Tester-Modes) and then trigger the error again? If you report the backtrace that appears below the error message, I can locate the problem and fix it.

**Reply by MassGaming25** [reply]
Hello, as requested the debug mode error:
![image](https://github.com/xenocrat/chyrp-lite/assets/102473883/7f58fe26-417c-4d99-8730-1e3c4b907db8)

**Comment by MassGaming25** [comment]
Got it working, just needed to enable multi-byte in the config. Thank you for the help!
